frontend/src/metabase-types/api/bookmark.ts

```diff
@@ -2,7 +2,13 @@ import type { CardId, CardType } from "./card";
 import type { CollectionId } from "./collection";
 import type { DashboardId } from "./dashboard";
 
-export const BOOKMARK_TYPES = ["card", "collection", "dashboard"] as const;
+export const BOOKMARK_TYPES = [
+  "card",
+  "collection",
+  "dashboard",
+  "snippet",
+  "indexed-entity",
+] as const;
```

Comment:
nit, not seeing how this relates to this refactor?

Reply:
If I don't add those, I get this:

```
frontend/src/metabase/collections/components/ActionMenu/ActionMenu.tsx:108:42 - error TS2345: Argument of type '"card" | "collection" | "dashboard" | "snippet" | "indexed-entity"' is not assignable to parameter of type '"card" | "collection" | "dashboard"'.
  Type '"snippet"' is not assignable to type '"card" | "collection" | "dashboard"'.

108     toggleBookmark?.(item.id.toString(), normalizeItemModel(item));
                                             ~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
my b, carry on then